feature_tests/src/lifetimes.rs

```diff
@@ -16,11 +16,12 @@ pub mod ffi {
         }
     }
 
-    impl<'b, 'a: 'b> Bar<'b, 'a> {
-        pub fn foo(&'b self) -> &'b Foo<'a> {
-            self.0
-        }
-    }
+    // TODO: add after #191 is resolved.
```

Comment:
must we remove this test if it's still testing JS stuff (and not breaking C++)?

 we can leave a comment that it's untested in C++

Reply:
We get 
```
thread 'main' panicked at 'not yet implemented: Returning references from Rust to C++ is not currently supported', tool/src/cpp/conversions.rs:177:13
```
When running `./regen_integration_tests.sh` when we have this test in, is that okay?

Reply:
Wait, did you readd it? Please remove it, I made this comment here assuming that the regen had been succeeding with this code previously.

(Our regen tests really need to be better to catch this)

Reply:
https://github.com/rust-diplomat/diplomat/pull/195

My bad. In retrospect this probably should've been a separate PR anyways.